source/agora/node/FullNode.d

```diff
@@ -322,7 +322,10 @@ public class FullNode : API
 
     protected bool acceptBlock(const ref Block block) @trusted
     {
-        return this.ledger.acceptBlock(block);
+        // Attempt to add block to the ledger (it may be there by other means)
+        this.ledger.acceptBlock(block);
```

Comment:
You reverted the `&&` ?

Reply:
Yes but updated the documentation to explain why.